src/Pages/CreateProduct/components/ProductForm/index.tsx

```diff
@@ -91,10 +120,7 @@ export const ProductForm = () => {
           >
             {t("form.buttons.submit")}
           </Button>
-          <Button
-            onClick={() => navigation(URL.PRODUCTS)}
-            style={{ marginLeft: "55px" }}
-          >
+          <Button onClick={onCancel} style={{ marginLeft: "55px" }}>
```

Comment:
Do not use `style` every, `sx` or `styled components`

Reply:
Fixed